docs/contributors/folder-structure.md

```diff
@@ -0,0 +1,192 @@
+# Folder Structure
+
+The following snippet explains how the WooCommerce Blocks repository is structured omitting irrelevant or obvious items with further explanations:
+
+    │
+    ├── LICENSE
+    ├── README.md
+    ├── .github/CODEOWNERS
+    ├── .github/CODE_OF_CONDUCT.md
+    ├── .github/CONTRIBUTING.md
+    ├── .github/SECURITY.md
+    │
+    ├── .distignore
+    ├── .editorconfig
+    ├── .eslintignore
+    ├── .eslintrc
+    ├── .gitattributes
+    ├── .gitignore
+    ├── .prettierrc.js
+    ├── .stylelintrc.json
+    ├── phpcs.xml.dist
+    │   Dot files and config files used to configure the various linting tools
+    │   used in the repository (PHP, JS, styles...).
+    │
+    ├── .nvmrc
+    │   Required Node version of the project.
+    │
+    ├── babel.config.js
+    ├── global.d.ts
+    ├── tsconfig.base.json
+    ├── tsconfig.json
+    ├── webpack.config.js
+    │   Transpilation and bundling config files.
+    │
+    ├── phpcs.xml
+    │   Config file for the PHP Coding Standards.
+    │
+    ├── phpunit.xml.dist
+    │   Config file for PHPUnit.
+    │
+    ├── postcss.config.js
+    │   Config file for PostCSS.
+    │
+    ├── renovate.json
+    │   Config file for Renovate.
+    │
+    ├── docker-compose.yml
+    │   Docker config files for the development and testing environment.
```

Comment:
Not sure we need this anymore (because we use wp-env).

Reply:
@nerrad Would you suggest removing this section from the documentation and creating an issue to remove `docker-compose.yml` from the repo?